src/App.tsx

```diff
@@ -76,18 +58,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !error && posts.length === 0 && (
```

Comment:
it's better to move such complicated conditions to variables to keep your jsx clean. pls fix everywhere